SUPPORT.md

```diff
@@ -38,11 +38,26 @@ If all of this sounds like we're putting you off, that's not how this is meant.
 
 ### Consider this
 
-```txt
-TODO:
-- clearly what the limits of community support are
-- which venue is approrpriate for which enquiry
-```
+Support for FluxCD users and community is mostly provided by volunteers, who are all on equal footing as peers in the FluxCD community. This implies that (free) support is provided on a best-effort basis, with no SLA or quality-of-service guarantee.
+
+[Best-effort delivery](https://en.wikipedia.org/wiki/Best-effort_delivery) is explained on Wikipedia in terms of a "**best-effort network**, [on which] all users obtain best-effort service. Under best-effort, network performance characteristics such as network delay and packet loss depend on the current network traffic load."
+
+Similar to a best-effort network, the capacity for our community as a whole to provide quality support for community members, and a welcoming environment for all contributors, depends heavily on the grace and good behavior of individual community members.
+
+You can help ensure a higher quality of best-effort support by formulating inquiries thoughtfully and making a considerate effort to place them in the most appropriate venue.
+
+Here are some guidelines about which venue makes the most sense:
+
+| Form of inquiry                                                      | Venue                 |
```

Comment:
Maybe add "quick question" as another form of inquiry, with the venue being "Slack".

Reply:
I added it at the bottom of the list